anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-ack.adoc

```diff
@@ -51,7 +51,14 @@ By default, the connector honors the time defined at the queue level (*Message L
 When using `acknowledgementTimeout`, consider expected app-time processing, including time for unexpected delays, such as external systems delay and app back-pressure due to high load. For example, if you expect the consumed message to be processed in 10 seconds, set `acknowledgementTimeout` to a minimum of 15 seconds.
 
 [IMPORTANT]
-When using the `prefetch` mode in the Anypoint MQ Subscriber source, the time that the message remains in the local buffer doesn’t count as part of the timeout. The `acknowledgementTimeout` timer begins when the message is dispatched to the flow, not when it is taken from the queue. The default TTL is two minutes. If you need a different TTL, you must define it explicitly in the `acknowledgementTimeout` parameter of the subscriber.
+----
+When using the `prefetch` mode in the Anypoint MQ Subscriber source, the time that the message remains in the local buffer doesn’t count as part of the timeout. The `acknowledgementTimeout` timer begins when the message is consumed. The default TTL is two minutes. If you need a different TTL, you must define it explicitly in the `acknowledgementTimeout` parameter of the subscriber.
+
+If the time that the message remains in the buffer reaches 80% of the timeout, the expiration time is extended.
```

Comment:
Taking into account this last sentence, shouldn't then the previous one be deleted?

Reply:
What about saying when using the `prefetch` mode in the Anypoint MQ Subscriber source,... and saying something like the timeout will try to be preserved. Like trying to make notice that this is not always true. 
That the timeout will try to be honoured when possible, and that is when 80% of time was reached in the internal buffer or it actually expired.

Reply:
See the change. I'm not sure we want to say the timeout isn't honored if it expired (do we?) Isn't that expected behavior?